change/@ni-nimble-components-46e8e11c-bca9-4ade-91b6-269da5bcce45.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I'm disabling this test in https://github.com/ni/nimble/pull/1180 due to the instability on main.

Hopefully this can be reproduced locally instead. If not then we should find a way to run this test without introducing failures. Such as changing the expect clauses and just doing logging. Searching through logs may be a challenge though. Open to other ideas that don't involve making main unstable.